### PR TITLE
Revendor containerd and update dependencies

### PR DESCRIPTION
**Pull request by kevpar**
This pulls in the big upstream merge PR from the containerd fork.

### CODE REVIEW

File: cmd/containerd/builtins_windows.go [hunk 1]
```diff
@@ -22,8 +22,6 @@ import (
 	_ "github.com/containerd/containerd/diff/lcow"
 	_ "github.com/containerd/containerd/diff/windows"
 	_ "github.com/containerd/containerd/runtime/v2"
-	_ "github.com/containerd/containerd/runtime/v2/runhcs"
```
Comment (jterry75) [inline cmd/containerd/builtins_windows.go]: So excited!

File: vendor.conf [hunk 2]
```diff
@@ -22,9 +22,9 @@ github.com/docker/go-units v0.3.1
 github.com/docker/spdystream 449fdfce4d962303d702fec724ef0ad181c92528
 github.com/emicklei/go-restful v2.2.1
 github.com/godbus/dbus v3
-github.com/gogo/googleapis 08a7655d27152912db7aaf4f983275eaf8d128ef
-github.com/gogo/protobuf v1.0.0
-github.com/golang/protobuf v1.1.0
+github.com/gogo/googleapis v1.2.0
```
Comment (jterry75) [inline vendor.conf]: These just match cd?

Comment (kevpar) [reply vendor.conf]: Yes, I updated a few vendorings but it was always to match containerd

Comment (jterry75) [reply vendor.conf]: Deal thanks

**Pull request merged**